app/src/main/java/com/example/habitsmasher/Habit.java

```diff
@@ -31,13 +32,14 @@ public Habit () {
      * @param habitId id of habit
      * @param habitEvents list holding habit events
      */
-    public Habit (String title, String reason, Date date, String days, long habitId, HabitEventList habitEvents) {
+    public Habit (String title, String reason, Date date, String days, boolean isPublic, long habitId, HabitEventList habitEvents) {
         super(habitId);
         _title = title;
         _reason = reason;
         _date = date;
         _days = days;
         _habitEvents = habitEvents;
+        _public = isPublic;
```

Comment:
Minor:  I feel like this field could be named something more descriptive like privacy or something

Reply:
Thought about it, only issue is that its vague what the boolean would represent, like what would `privacy = true` mean? `isPublic` sounds not great but it at least is understandable

Reply:
I agree, I think isPublic makes the most sense